src/components/password/components.test-pw.tsx

```diff
@@ -0,0 +1,165 @@
+            label="Password - readOnly"
+            value="Password"
+            readOnly
+            {...{ [validationType]: true }}
+            mb={2}
+          />
+        </div>
+      ))}
+    </Box>
+  );
+};
+
+export const PasswordValidationsAsAString = () => {
+  return (
+    <Box>
+      {VALIDATIONS.map((validationType) => (
+        <div key={`${validationType}-string-component`}>
+          <Password
+            label="Password"
+            value="Password"
+            {...{ [validationType]: "Message" }}
+            mb={2}
+          />
+          <Password
+            label="Password - readOnly"
+            value="Password"
+            readOnly
+            {...{ [validationType]: "Message" }}
+            mb={2}
+          />
+        </div>
+      ))}
+    </Box>
+  );
+};
+
+export const PasswordValidationsAsAStringWithTooltipCustom = () => {
+  return (
+    <Box>
+      {VALIDATIONS.map((validationType) => (
+        <div key={`${validationType}-string-component`}>
+          <Password
+            label="Password"
+            value="Password"
+            {...{ [validationType]: "Message" }}
+            mb={2}
+            tooltipPosition="bottom"
+          />
+        </div>
+      ))}
+    </Box>
+  );
+};
+
+export const PasswordValidationsAsAStringDisplayedOnLabel = () => {
+  return (
+    <Box>
+      {VALIDATIONS.map((validationType) => (
+        <div key={`${validationType}-string-label`}>
+          <Password
+            label="Password"
+            value="Password"
+            validationOnLabel
+            {...{ [validationType]: "Message" }}
+            mb={2}
+          />
+          <Password
+            label="Password - readOnly"
+            value="Password"
+            validationOnLabel
+            readOnly
+            {...{ [validationType]: "Message" }}
+            mb={2}
+          />
+        </div>
+      ))}
+    </Box>
+  );
+};
+
+export const PasswordNewDesignsValidation = () => {
+  return (
+    <Box>
+      <CarbonProvider validationRedesignOptIn>
+        {(["error", "warning"] as const).map((validationType) =>
+          SIZES.map((size) => (
+            <div style={{ width: "296px" }} key={`${validationType}-${size}`}>
+              <Password
+                m={4}
+                label={`${size} - ${validationType}`}
+                defaultValue="Password"
+                labelHelp="Hint text (optional)"
+                size={size}
+                {...{ [validationType]: "Message" }}
+              />
+              <Password
+                m={4}
+                label={`readOnly - ${size} - ${validationType}`}
+                defaultValue="Password"
+                size={size}
+                labelHelp="Hint text (optional)"
+                readOnly
+                {...{ [validationType]: "Message" }}
+              />
+            </div>
+          ))
+        )}
+      </CarbonProvider>
+    </Box>
+  );
+};
+
+export default PasswordComponent;
```

Comment:
Is a default export needed? PasswordComponent could be exported the same as the other examples.

Reply:
Good shout